### ISSUE TITLE
Allow more customization

### ISSUE THREAD

**Issue by rosenpin** [rosenpin/AlwaysOnDisplayAmoled#67]
- [ ] More watch faces
- [ ] Custom background/wallpapers

**Comment by jdrescher2006** [comment]
I have some more ideas from an application which I really miss the most on android: Noka-Sleeping-Screen.
- **custom font** (Nokia-Sleeping-Screen used a dotted font. This would further reduce the battery drain because just a few dots where lighted)
- **change color of font for night mode** (this is also known from Nokia-Sleeping-Screen. In day mode they would use white color, while in night mode red color)

Screenshot of Nokia-Sleeping-Screen in night mode:
[http://www.nokiabreak.com/wp-content/uploads/2012/09/Nokia-Sleeping-Screen-320x180.jpg]

**Comment by rosenpin** [comment]
> custom font 
The app already has custom font option, I'll add a large variety in a future update though.

> change color of font for night mode
I added it to the list

Thanks for the suggestions.

**Comment by rosenpin** [comment]
@jdrescher2006
I've added the font you requested in the latest version.
It will be available in the Play Store today or tomorrow.

**Comment by jdrescher2006** [comment]
Thanks for that, I will try it out.
Anyway you seem to be a real workaholic, working on the app day and night :+1:

**Comment by rosenpin** [comment]
Added 3 new watchfaces

![classic](http://i.imgur.com/vx5X1Hb.png)

![pebble style](http://i.imgur.com/DBJiZrO.png)

![s7 style](http://i.imgur.com/UtU30Wo.png)

**Comment by adiasandre96** [comment]
I have a suggestion: What if we could choose which app notifications should be shown?

**Comment by rosenpin** [comment]
@adiasandre96 
Do you mean a filter/blacklist apps notifications?

**Comment by adiasandre96** [comment]
@rosenpin 
Exactly! For example, I just want to bring up the notification of missed calls and received messages. All other I do not want to appear. It could be two options (where only one of them could be selected): "Show only" and "Hide only".
It's just a suggestion that I think would be helpful. At this time the application is already great but an improvement is always welcome!
